packages/react-examples/src/charting/VerticalBarChart/VerticalBarChart.Basic.Example.tsx

```diff
@@ -37,39 +37,39 @@ export class VerticalBarChartBasicExample extends React.Component<IVerticalBarCh
       {
         x: 40000,
         y: 13000,
-        legend: 'seventh',
+        legend: 'Fourth',
```

Comment:
Might be better to change these labels to something non-numeric so that it doesn't turn into nonsense if the order changes or things are added